src/components/Users/UserResetPassword.tsx

```
@@ -1,216 +1,219 @@
+                        type="password"
+                        onChange={(e) => {
+                          field.onChange(e.target.value);
+                        }}
+                        onFocus={() => setIsPasswordFieldFocused(true)}
+                        onBlur={() => setIsPasswordFieldFocused(false)}
+                      />
+                    </FormControl>
+                    {form.formState.errors?.new_password_1?.message &&
+                      form.formState.errors?.new_password_1?.message.includes(
+                        t("new_password_same_as_old"),
+                      ) && <FormMessage />}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)